assets/articles/0048-webpack-plugin-written-by-chatgpt/webpack-plugin-written-by-chatgpt.adoc

```diff
@@ -0,0 +1,465 @@
+= webpack Plugin written by ChatGPT
+
+== Introduction
+
+No matter what social media platform you are on if you follow the tech space, or any space for that matter, you are most likely seeing content about ChatGPT or even content generated by ChatGPT.
+
+When it first came out a few weeks ago, I played with it, thought it was pretty cool and had my kids started using it to write guided stories with ideas from their imagination. When combined with MidJourney AI generated art the overall experience was quite amazing.
+
+Last night seeing https://dev.to/scriptedalchemy[@scriptedalchemy] share about his https://www.linkedin.com/feed/update/urn:li:activity:7009395526903234560/[experiences] with it I decided to try something a little more challenging.
```

Comment:
Links should be opened on the new tab

Reply:
Cmd + Click :D Or mouse wheel click?

Reply:
or add to the link "^" attribute :D  example: [experiences^]